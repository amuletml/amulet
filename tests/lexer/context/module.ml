(* Implicit begin *)
module X =
  let _ = 1

(* Explicit begin *)
module X = struct
  let _ = 1
end

(* Explicit begin with weird alignment *)
module X =
struct
  let _ = 1
end

module X =
  struct
    let _ = 1
  end

module X = Y

(* Access modifiers *)
private module Y =
  let a = 0
