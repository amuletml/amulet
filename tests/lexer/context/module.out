
module X =
  $begin let _ = 1


$end $end $sep module X = struct
  let _ = 1
$end end


$sep module X =
struct
  let _ = 1
$end end

$sep module X =
  struct
    let _ = 1
  $end end

$sep module X = Y


$sep private module Y =
  $begin let a = 0
$end $end <eof>
